src/SFML/Graphics/Text.cpp

```diff
@@ -231,6 +294,11 @@ void Text::draw(RenderTarget& target, RenderStates states) const
 
         states.transform *= getTransform();
         states.texture = &m_font->getTexture(m_characterSize);
+
+        // Only draw the outline if there is something to draw
+        if (m_outlineThickness > 0)
```

Comment:
I think this can be shortend to `if (m_outlineThickness)` because `m_outlineThickness` is an `unsigned int`.